src/api/gitHub.js

```diff
@@ -290,11 +221,114 @@ export const determineGraphQLEndpoint = async (server) => {
   }
 };
 
+
+/**
+ * fetch options for fetchRepoInOrg
+ *
+ * @param {string} org the organization
+ * @param {string} token the token
+ * @param {string} cursor the last repository fetched
+ * @param {boolean} selfsigned the selfsigned option 
+ * @returns {object} the fetch options
+ */
+export const fetchRepoInOrgInfoOptions = async (org, token, cursor, selfsigned) => {
+  let fetchOptions = {
+    method: "POST",
+    headers: {
+      Authorization: `bearer ${token}`,
+    },
+    body: JSON.stringify({
+      query: `{
+        rateLimit {
+          limit
+          cost
+          remaining
+          resetAt
+        }
+        organization(login: "${org}") {
+          repositories(first: 50${cursor}){
+            totalCount
+            edges {
+              cursor
+              node {
+                projects(first:1){
+                  totalCount
+                }  
+                hasWikiEnabled
+                issues(first: 1) {
+                  totalCount
+                }
+                pullRequests(first: 1) {
+                  totalCount
+                }
+                name
+                id
+                url
+                isPrivate
+              }
+            }
+          }
+        }
+      }`,
+    }),
+  }
+  if (selfsigned) {
+    /**
+    * set agent to a new https.Agent if it hasn't alredy been set
+    */
+    if (Object.entries(agent).length === 0) {
+      agent = new https.Agent({rejectUnauthorized: false,});
+    }
+    fetchOptions.agent = agent
+  }
+  return fetchOptions
+};
+
+
+/**
+ * fetch options for fetchOrgInfo
+ * 
+ * @param {string} org the org
+ * @param {string} token the token
+ * @param {boolean} selfsigned the selfsigned option 
+ * @returns {object} the fetch options
+ */
+export const fetchOrgInfoOptions = async (org, token, selfsigned) => {
```

Comment:
This doesn't need to be async because it doesn't use promises anywhere. You can remove this, and then avoid the await when you call the function:

```suggestion
export const fetchOrgInfoOptions = (org, token, selfsigned) => {
```

Reply:
I changed the logic as suggested.

I used async for consistency as ```determineGraphQLEndpoint``` was using async.